torch/nn/functional.py

```diff
@@ -1192,3 +1191,6 @@ def normalize(input, p=2, dim=1, eps=1e-12):
         eps (float): small value to avoid division by zero. Default: 1e-12
     """
     return input / input.norm(p, dim, True).clamp(min=eps).expand_as(input)
+
+def im2col(input, kH, kW, dH, dW, padH, padW, sH, sW):
```

Comment:
It would be nicer to use the same param names we use for all Conv functions + accept scalars or tuples for `kernel_size`, `stride`, etc. I'd also change this function to `unfold` and make it work for arbitrary number of dims (expect `len(kernel_size) + 2` dims). it can raise an error for `len(kernel_size) != 2` for now

Reply:
Param names changed to match conv functions